tests/sentry/ratelimits/utils/test_get_rate_limit_value.py

```diff
@@ -49,10 +49,42 @@ class TestEndpoint(Endpoint):
             20, 4
         )
 
+    def test_inherit(self):
+        class ParentEndpoint(Endpoint):
+            rate_limits = {"GET": {RateLimitCategory.IP: RateLimit(100, 5)}}
+
+        class ChildEndpoint(ParentEndpoint):
+            rate_limits = {"GET": {}}
+
+        assert get_rate_limit_value("GET", ChildEndpoint, RateLimitCategory.IP) == RateLimit(100, 5)
+
+    def test_multiple_inheritance(self):
+        class ParentEndpoint(Endpoint):
+            rate_limits = {"GET": {RateLimitCategory.IP: RateLimit(100, 5)}}
+
+        class Mixin:
+            rate_limits = {"GET": {RateLimitCategory.IP: RateLimit(2, 4)}}
+
+        class ChildEndpoint(ParentEndpoint, Mixin):
+            rate_limits = {"GET": {}}
+
+        assert get_rate_limit_value("GET", ChildEndpoint, RateLimitCategory.IP) == RateLimit(100, 5)
+
     def test_non_endpoint(self):
         """Views that don't inherit Endpoint should not return a value."""
 
         class TestEndpoint:
             pass
 
-        assert get_rate_limit_value("GET", TestEndpoint, RateLimitCategory.IP) is None
+        assert (
+            get_rate_limit_value("GET", TestEndpoint, RateLimitCategory.IP)
+            == settings.SENTRY_RATELIMITER_DEFAULTS[RateLimitCategory.IP]
```

Comment:
I don't think this is correct. We don't want any rate limit returned for a class that doesn't inherit endpoint.